lib/parts/auths/apple.dart

```diff
@@ -26,7 +28,7 @@ class AppleAuth extends AuthProvider {
       );
 
       if (session != null) {
-        var res = await application.api.post(_fetchCallbackRoute, {
+        var res = await _api.post(_fetchCallbackRoute, {
```

Comment:
This is ok as long as _api comes from _AuthApple settings class. Remember to still use the same application.api if _api is null

Reply:
ready